app/src/main/java/me/hufman/androidautoidrive/phoneui/fragments/SpotifyApiErrorDialog.kt

```diff
@@ -1,18 +1,76 @@
 package me.hufman.androidautoidrive.phoneui.fragments
 
+import android.app.Activity
 import android.app.AlertDialog
 import android.app.Dialog
+import android.content.Intent
 import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
 import androidx.fragment.app.DialogFragment
 import me.hufman.androidautoidrive.R
-import java.lang.IllegalStateException
+import me.hufman.androidautoidrive.phoneui.SpotifyAuthorizationActivity
+import me.hufman.androidautoidrive.phoneui.visible
 
 class SpotifyApiErrorDialog: DialogFragment() {
 	companion object {
+		const val TAG = "SpotifyApiErrorDialog"
 		const val EXTRA_CLASSNAME = "classname"
```

Comment:
Thank you for the refactor to a full Dialog with a title and message! Would you like to slip in an extra refactor to change the CLASSNAME to a TITLE to match the viewmodel?

Reply:
You're welcome! Changed CLASSNAME to TITLE throughout the dialog